composer.json

```diff
@@ -37,7 +37,7 @@
     },
     "require-dev": {
         "squizlabs/php_codesniffer": "^2.0",
-        "symfony/phpunit-bridge": "^3.3"
+        "symfony/phpunit-bridge": "^3.4.26 || ^4.2"
```

Comment:
i tend to only require ^5.0 of the bridge. it is only a tool to run tests, the version does not matter for the actual code so we don't need to support a wide range.